### PR TITLE
refactor: add notifications in utils

### PR DESCRIPTION
**Pull request by xyc**
### What does this PR do?
Refactor: create notification service in utils-vscode

### What issues does this PR fix or reference?
@W-8183073@

### CODE REVIEW

File: packages/salesforcedx-utils-vscode/src/commands/notificationService.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Observable } from 'rxjs/Observable';
+import { workspace } from 'vscode';
```
Comment (lcampos) [inline packages/salesforcedx-utils-vscode/src/commands/notificationService.ts:9]: lets merge these and have only one import for `vscode`

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStop.ts [hunk 2]
```diff
@@ -14,7 +17,7 @@ import { showError } from './commandUtils';
 const sfdxCoreExports = vscode.extensions.getExtension(
   'salesforce.salesforcedx-vscode-core'
 )!.exports;
-const { notificationService, telemetryService } = sfdxCoreExports;
+const { telemetryService } = sfdxCoreExports;
```
Comment (lcampos) [inline packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStop.ts:20]: Do we have another work item to finish replacing the rest of the telemetry service usage ?

File: packages/salesforcedx-utils-vscode/test/unit/commands/progressNotification.test.ts [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import * as proxyquire from 'proxyquire';
+import { Subject } from 'rxjs/Subject';
+import * as sinon from 'sinon';
```
Comment (lcampos) [inline packages/salesforcedx-utils-vscode/test/unit/commands/progressNotification.test.ts:11]: Let's merge these two imports for `sinon`

**Review by lcampos — APPROVED** [review]
Left a couple of comments on imports, everything else looks good to get merged

File: packages/salesforcedx-utils-vscode/test/unit/commands/progressNotification.test.ts (hunk 3 above)
Comment (xyc) [reply packages/salesforcedx-utils-vscode/test/unit/commands/progressNotification.test.ts]: Updated - see https://github.com/forcedotcom/salesforcedx-vscode/pull/2660/commits/1b4a811e1eaf5e8e52575b8d550724bd598664ed

File: packages/salesforcedx-utils-vscode/src/commands/notificationService.ts (hunk 1 above)
Comment (xyc) [reply packages/salesforcedx-utils-vscode/src/commands/notificationService.ts]: Updated - see https://github.com/forcedotcom/salesforcedx-vscode/pull/2660/commits/1b4a811e1eaf5e8e52575b8d550724bd598664ed

File: packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStop.ts (hunk 2 above)
Comment (xyc) [reply packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcStop.ts:20]: Created W-8290977 for replacing the rest of telemetryService

**Pull request merged**